app/views/layouts/index.njk

```diff
@@ -9,11 +9,18 @@
 
     <div class="govuk-grid-column-one-half">
       <h2 class="govuk-heading-m">Components</h2>
-      <ul class="govuk-list">
+      <dl class="govuk-list">
```

Comment:
I'd be tempted to do away with the `dl` and structure of having the primary example as the definition term and the secondary examples as definition descriptions as it's creating a hierarchy that I'm not sure is there. I'd probably go for a heading for the component name (no anchor element) and then a list of all the examples. Although the first example is possibly going to be the most used I don't think it's necessarily the best example.

Alternatively if you do want to keep the `dl` I'd do away with the `ul` as a `dt` can have multiple `dd`s so I've just make all the other examples `dd`s

Reply:
heading and list sounds good to me, I'll update it, thank you @timsb !

Reply:
@timsb I have updated with your suggestion and I saw there was also something else that was possibly partially implemented around displaying a component's readme that I figured I might as well implement at the same time

<img width="788" alt="CleanShot 2021-08-09 at 12 39 19@2x" src="https://user-images.githubusercontent.com/100650/128700672-9b245c67-e77c-4235-be11-3897552c94ba.png">

If you click view the readme it takes you to `/components/:component` and I've added in the needed code to render the readme at this route. From a comment it seemed like this was already an intention that hadn't been implemented. Small affordance but maybe helpful and it was quick to add! I can remove it if you think it's not useful 🤷‍♂️

Reply:
Yeah I'm happy to link to READMEs. I think in a lot of places they're pretty sparse but hopefully this will encourage us/me to expand on them and make them more useful.
The markup changes look good to me as well.